src/vcbuild/dmd.vcxproj

```diff
@@ -216,6 +218,13 @@
   <ItemGroup>
     <ClInclude Include="../dmd/**/*.h" />
   </ItemGroup>
+  <ItemGroup>
+    <CustomBuild Include="..\..\VERSION">
+      <Message>Creating VERSION...</Message>
+      <Command>$(_rdmdExe) ..\..\config.d $(IntDir)generated ..\..\VERSION</Command>
+      <Outputs>$(IntDir)generated\VERSION;%(Outputs)</Outputs>

```

Comment:
```
fatal error LNK1112: module machine type 'x86' conflicts with target machine type 'x64' [D:\a\1\s\src\vcbuild\dmd.vcxproj]
```

Do we need to set the architecture here too?
BTW if using `rdmd` is a problem, `dmd -run` (and `ldmd2 -run`) might be an alternative?

Reply:
> Do we need to set the architecture here too?

Yes, unfortunately LDC cannot build an x64 executable if the environment is configured for the x86 MS C++ compiler (and vice versa).